.gitignore

```diff
@@ -15,3 +15,6 @@ ehthumbs.db
 Thumbs.db
 
 storybook-static
+
+library.json
+semantics.json
```

Comment:
dont we need semantics.json in git?

Reply:
Buuut, some of the workflows we use need it for now, so we'll undo this change.